NAS-102125 add docs for disk type, rotation rate, and model (#1208)
freenas/freenas-docs: userguide/storage.rst

:guilabel:`Available Disks`, type :literal:`^(da)|(nvd)` in
:guilabel:`Filter disks by name`.

> The type and maximum capacity is dislayed for available disks.

Reviewer: ```suggestion
Type and maximum capacity is displayed for available disks.
```

> The *Rotation Rate*, *Model*, and *Serial* of an available disk can be

Reviewer: ```suggestion
To show the disk *Rotation Rate*, *Model*, and *Serial*, click
```

> displayed by clicking |ui-chevron-right|.

Reviewer: ```suggestion
|ui-chevron-right|.
```
